packages/mask/src/plugins/ITO/SNSAdaptor/ITO.tsx

```diff
@@ -458,16 +461,15 @@ export function ITO(props: ITO_Props) {
             return t('plugin_ito_out_of_stock_hit')
         }
 
-        const _text =
-            Number(availability?.swapped) > 0
-                ? t('plugin_ito_your_swapped_amount', {
-                      amount: formatBalance(availability?.swapped ?? 0, token.decimals),
-                      symbol: token.symbol,
-                  })
-                : t('plugin_ito_your_claimed_amount', {
-                      amount: formatBalance(tradeInfo?.buyInfo?.amount_bought ?? 0, token.decimals),
-                      symbol: token.symbol,
-                  })
+        const _text = new BigNumber(availability?.swapped ?? '0').isGreaterThan(0)
+            ? t('plugin_ito_your_swapped_amount', {
+                  amount: formatBalance(availability?.swapped ?? 0, token.decimals),
+                  symbol: token.symbol,
+              })
+            : t('plugin_ito_your_claimed_amount', {
+                  amount: formatBalance(tradeInfo?.buyInfo?.amount_bought ?? 0, token.decimals),
+                  symbol: token.symbol,
+              })
```

Comment:
```suggestion
        const _text = new BigNumber(availability?.swapped || 0).isGreaterThan(0)
            ? t('plugin_ito_your_swapped_amount', {
                  amount: formatBalance(availability?.swapped || 0, token.decimals),
                  symbol: token.symbol,
              })
            : t('plugin_ito_your_claimed_amount', {
                  amount: formatBalance(tradeInfo?.buyInfo?.amount_bought || 0, token.decimals),
                  symbol: token.symbol,
              })
```

`||` is safer than `??` here. `||` can circumvent `availability?.swapped === ''`.
Besides, `??` will be compiled into much more code than `||`.